Sources/Swift/SentryExperimentalOptions.swift

```diff
@@ -1,7 +1,18 @@
 @objcMembers
 public class SentryExperimentalOptions: NSObject {
+    /**
+     * Disables swizzling of`NSData` to automatically track file operations.
+     *
+     * - Note: Swizzling is enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
+     *         This option allows you to disable swizzling for `NSData` only, while keeping swizzling enabled for other classes.
+     *         This is useful if you want to use manual tracing for file operations.
+     */
+    public var disableDataSwizzling = false
+
     /**
      * Enables swizzling of`NSFileManager` to automatically track file operations.
+     *
+     * - Requires: Swizzling must be enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
```

Comment:
I think we'd also need this in the headerdoc for `disableDataSwizzling`?

Reply:
Can you please elaborate how the current note not sufficient?

Reply:
I think @armcknight point is that you should also add this information to the headerdoc of `disableDataSwizzling`, because the same applies for `disableDataSwizzling`.